src/jiradeps/graphvizdeps.py

```diff
@@ -10,13 +9,16 @@
 import graphviz as gv
 import networkx as nx
 
-from .jirawrapper import (get_sprints, get_story_points, get_related_keys,
-                          get_team)
+from .jirawrapper import (get_sprints, get_story_points, get_related_keys, get_team)
 
 
-def visualize_with_graphviz(nx_graph: nx.DiGraph, epics,
-                            file_name=None, file_type='svg', group_epics=False,
-                            align_sprints=False, show_status=False):
+def visualize_with_graphviz(nx_graph: nx.DiGraph,
+                            epics,
+                            file_name=None,
+                            file_type='svg',
+                            group_epics=False,
+                            align_sprints=False,
+                            show_status=False):
```

Comment:
Strange, I think the original indentation was already PEP8 compliant. The new one doesn't look like an improvement.